### PR TITLE
Show link leading to the Food Preferences Wiki page.

### PR DESCRIPTION
**Pull request by tt-rasm**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes issue #8023 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Added two strings to the "pets.json" file, "foodWikiUrl" and "foodWikiText", and added the link into the "pets.jade" template.
screenshot 2016-12-08 11 16 55

[//]: # (Put User ID in here - found in Settings -> API)
----
UUID: 78059605-3801-41dc-adce-c0044af741c0

### CODE REVIEW

**Comment by khipkin** [comment]
I don't know what that yarn.lock file is, but I suspect it shouldn't be committed.

**Comment by tt-rasm** [comment]
@khipkin yarn.lock is just a lock file for people who use the [yarn package manager](https://yarnpkg.com/en/docs/yarn-lock).  I can remove it and re-commit if needed.

**Comment by Alys** [comment]
@Tallestthomas Yes, please remove that file.

**Comment by TheHollidayInn** [comment]
@Tallestthomas PR looks good. 

Side note: Did you need Yarn for this project? We will eventually need to commit that file in. Just not for this PR.

**Comment by tt-rasm** [comment]
@TheHollidayInn Didn't really _need_ Yarn, but I've switched to it almost exclusively because it tends to install dependencies quicker than npm which is nice, other than that they're pretty interchangeable.

**Comment by TheHollidayInn** [comment]
@Tallestthomas Sounds good. If you have a PR after next week and use Yarn, we will probably commit that file in.

**Comment by Alys** [comment]
@Tallestthomas Can you please remove yarn.lock from the .gitignore file too? If you need advice about other ways of ignoring files, you can ask here.

If we do start using yarn for this project (which I'm fine with if the dev team agrees), then we'd add all relevant config at the same time so that a single PR contained all related changes (that's best future audit/review purposes). At the moment, the yarn lock file is specific to your install and we don't use the .gitignore file for files like that - it would become large and difficult to maintain if we did. :)

**Comment by tt-rasm** [comment]
@Alys I've removed the yarn.lock from my .gitignore locally, repo shows its already been removed (probably due to another merge)

File: website/client-old/css/inventory.styl [hunk 1]
```diff
@@ -67,6 +67,10 @@ menu.pets div
   z-index: 1
   padding-bottom: 0
 
+  label
+    display:block
+
```
Comment (khipkin) [inline website/client-old/css/inventory.styl]: Nit: remove extra newlines.

File: website/client-old/css/inventory.styl [hunk 2]
```diff
@@ -67,9 +67,6 @@ menu.pets div
   z-index: 1
   padding-bottom: 0
 
-  label
```
Comment (khipkin) [inline website/client-old/css/inventory.styl]: @Tallestthomas Why did you remove these lines too?

Comment (tt-rasm) [reply website/client-old/css/inventory.styl]: @khipkin after retesting, they didn't actually do anything

Comment (khipkin) [reply website/client-old/css/inventory.styl]: Oh okay, that's a good reason. 😛

**Comment by TheHollidayInn** [comment]
Added towards your tier. Congrats on Tier 1!
